dmutils/email/exceptions.py

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+"""Digital Marketplace normalised exception classes."""
+
+
+class DMEmailApiError(Exception):
```

Comment:
I'd call this `EmailError` and I don't think we need separate classes form Mandrill and Notify. I don't think they'll overlap in code and having the same exception makes it easier to switch.